apps/docs/src/pages/components/action-bar.mdx

```diff
@@ -0,0 +1,48 @@
+import { Callout } from "nextra/components";
+import {
+  actionBarClasses,
+  HvActionBar,
+} from "@hitachivantara/uikit-react-core";
+
+import Playground from "@docs/components/code/Playground";
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData("ActionBar", "core", actionBarClasses);
+  return { props: { ssg: { meta } } };
+};
+
+<Description />
+
+<Page>
+
+### Usage
+
+```tsx live
+<HvActionBar style={{ gap: theme.space.xs }}>
+  <HvButton variant="secondaryGhost">Cancel</HvButton>
+  <div style={{ flex: 1 }} aria-hidden="true">
+    &nbsp;
+  </div>
+  <HvButton variant="secondaryGhost">Save</HvButton>
+  <HvDropDownMenu
+    dataList={[
+      { id: "export", label: "Export", icon: <Open />, disabled: true },
+      {
+        id: "preview",
+        label: <HvTypography variant="label">Preview</HvTypography>,
+        icon: <Preview />,
+      },
```

Comment:
Can we simplify this? We're showing ALL of the `HvDropDownMenu` API in a concise way, but it's not relevant for
```suggestion
      { id: "export", label: "Export" },
      { id: "preview", label: "Preview" },
```